rholang/src/main/scala/coop/rchain/rholang/interpreter/SystemProcesses.scala

```diff
@@ -21,7 +21,7 @@ object SystemProcesses {
   private val prettyPrinter = PrettyPrinter()
 
   // because only we do installs
-  private implicit val INSTALLS_MATCH_PHLOS = matchListQuote(CostAccount(Integer.MAX_VALUE))
+  private implicit val INSTALLS_MATCH_PHLOS = matchListQuote(Cost(Integer.MAX_VALUE))
```

Comment:
Costs.MAX_VALUE?

Reply:
TBH I didn't introduce constant in prod code on purpose - so the usage of `Integer.MAX_VALUE` is explicit and purposeful.